global.json

```diff
@@ -1,3 +1,3 @@
 {
-    "sources": [ "src" ]
+    "sources": [ "src", "../NuGet.Versioning/src" ]
```

Comment:
Yeah... but it's kinda necessary. In theory, it shouldn't break things if you have just cloned NuGet.Versioning.